network-api/networkapi/templates/buyersguide/pages/consumer_creepometer_page_2023.html

```diff
@@ -40,7 +40,7 @@ <h1 class="tw-font-normal tw-font-zilla tw-text-[48px] tw-leading-[44px] medium:
   </div>
 
   {% comment %} Quiz Section {% endcomment %}
```

Comment:
I appreciate you separating these sections using `{% comment %}`. 

In previous PR's of mine, I have tried to show separation of different sections by using `ID` or classnames, however this seems like a better idea to me as it shows the separation of sections in a cleaner, more readable manner. Plus, adding ID's to an element when we were not using the ID for anything else such as JS, felt strange to me.

Definitely going to keep this in mind moving forward!

Reply:
Thanks @danielfmiranda ! I still need to read upon best practices around identifying sections/blocks without creating new id/class solely for that purpose. This wasn't an issue before as we had to create id/class for SCSS to kick in. Now that we are using Tailwind, I do find debugging on dev tool very challenging because what we have now is a series of Tailwind utility classes 😱